test/functional/legacy/011_autocommands_spec.lua

```diff
@@ -67,10 +67,15 @@ describe('file reading, writing and bufnew and filter autocommands', function()
 
     it('FileReadPost (using gzip)', function()
       prepare_gz_file('Xtestfile', text1)
-      feed_command('let $GZIP = ""')
       --execute('au FileChangedShell * echo "caught FileChangedShell"')
       feed_command('set bin')
-      feed_command("au FileReadPost    *.gz   '[,']!gzip -d")
+      if iswin() then
+          feed_command('let $GZIP = ""')
```

Comment:
`call setenv('GZIP', v:null)` for both.

Reply:
Unfortunately, it does not work for me.
So I cannot accept the suggestion.

```
stack traceback:
	test/helpers.lua:73: in function 'expect'
	test/functional/legacy/011_autocommands_spec.lua:76: in function <test/functional/legacy/011_autocommands_spec.lua:68>


 1 ERROR
-- Tests exited non-zero: 1
-- Output to stderr:
gzip: warning: GZIP environment variable is deprecated; use an alias or script
Xtestfile:	 73.9% -- replaced with Xtestfile.gz
gzip: warning: GZIP environment variable is deprecated; use an alias or script
Xtestfile:	 73.9% -- replaced with Xtestfile.gz
gzip: warning: GZIP environment variable is deprecated; use an alias or script
Xtestfile:	 73.9% -- replaced with Xtestfile.gz
```

Reply:
Does your shell set `GZIP`? Does the problem remain if you change the shell to `sh`?

Reply:
OH, it fixes my problem!
Thanks.